src/externalCaller/parserCaller.ts

```diff
@@ -1,5 +1,13 @@
 import fetch from 'node-fetch';
 import config from '../config';
+import { setTimeout } from 'timers/promises';
+
+export type PartialGetItemResponse = {
```

Comment:
This makes sense but looking at this against the type below makes me want to double check the casing of `item_id` (which is `itemId` below).

Reply:
👍 yep the response from the parser is item_id, but snake_casing isn't idiomatic JS/TS so that's why it's always itemId in const vars/etc.

Reply:
I understand -- here we have 2 types one called `PartialGetItemResponse` the other called `ItemResponse` just below it where one sounds like a Partial of the other (even if it isn't) and they each have the same property in snake/camel variants.